fsharp-backend/tests/Tests/Serialization.Tests.fs

```diff
@@ -606,15 +624,15 @@ module GenericSerializersTests =
       // LibExecution
       // ------------------
 
-      v<CT.Dval.T> "complete" testClientDval
+      v<CTRuntime.Dval.T> "complete" testClientDval
 
       v<LibExecution.DvalReprInternalNew.RoundtrippableSerializationFormatV0.Dval>
         "complete"
         (testDval
          |> LibExecution.DvalReprInternalNew.RoundtrippableSerializationFormatV0.fromRT)
 
-      v<LibExecution.ProgramTypes.Oplist> "complete" testOplist
-      v<LibExecution.ProgramTypes.Handler.T> "simple" testHttpHandler
+      v<LibExecution.ProgramTypes.Oplist> "complete" ProgramTypes.testOplist
+      v<LibExecution.ProgramTypes.Handler.T> "simple" ProgramTypes.testHttpHandler
```

Comment:
Surely this isn't allowed serialize anymore?

Reply:
It turns out they are.
- `LibExecution.ProgramTypes.Handler.T` seems to be used in a canvas-clone test, and the .allow is registered in `Tests.fs` just for that
- `LibExecution.ProgramTypes.Oplist` is used in `Canvas`'s `loadJsonFromDisk`


As I look through other non-ClientTypes types, I seem to find similar edge cases.